src/components/Board/index.js

```diff
@@ -1,35 +1,42 @@
 import React from 'react';
+import './styles.css';
 import Square from '../Square';
 
-const Board = ({ squares, onClick}) => {
+const Board = ({ squares, onClick, winnerLine}) => {
 
-    const renderSquare = i => {
-        return <Square
-                    value={squares[i]}
-                    onClick={() => onClick(i)}
-                />
+    const determineWinnerSquare = (i, winnerLine) => {
```

Comment:
Functions that return a boolean value are usually named with an `is` prefix, like asking a question: `isWinnerSquare?` would be a better name for it (without the ? of course)